modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/eventsDefinitions.js

```diff
@@ -29,29 +28,6 @@ export const IS_LOADING_MODAL = 'is-loading-modal';
 export const OPEN_MODAL = 'open-modal';
 export const OPEN_SIDE_PANEL = 'open-side-panel';
 export const PRODUCT_OPTIONS_CHANGED = 'product-options-changed';
-export const PRODUCT_REMOVED = 'productRemovedFromCart';
+export const PRODUCT_REMOVED_FROM_CART = 'product-removed-from-cart';
 export const SIDE_PANEL_CLOSED = 'side-panel-closed';
 export const UPDATE_DATASET_DISPLAY = 'update-data-set-display';
-
-export default {
-	ADD_TO_ORDER,
-	AUTOCOMPLETE_VALUE_UPDATED,
-	CART_PRODUCT_QUANTITY_CHANGED,
-	CHANGE_ACCOUNT,
-	CHANGE_ORDER,
-	CLOSE_MODAL,
-	CLOSE_SIDE_PANEL,
-	CP_INSTANCE_CHANGED,
-	CURRENT_PRODUCT_STATUS_CHANGED,
-	CURRENT_PRODUCT_UPDATED,
-	DATASET_ACTION_PERFORMED,
-	DATASET_DISPLAY_UPDATED,
-	IFRAME_LOADED,
-	IS_LOADING_MODAL,
-	OPEN_MODAL,
-	OPEN_SIDE_PANEL,
-	PRODUCT_OPTIONS_CHANGED,
-	PRODUCT_REMOVED,
-	SIDE_PANEL_CLOSED,
-	UPDATE_DATASET_DISPLAY,
-};
```

Comment:
This is dangerous. In JSP's we're using `eventsDefinitions` and I'm afraid sometimes we're using it as a `default` import. Did you double check?